app/assets/javascripts/app/views/aspect_membership_view.js

```diff
@@ -13,8 +13,7 @@ app.views.AspectMembership = app.views.AspectsDropdown.extend({
 
   events: {
     "click ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
-    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
-    "click ul.aspect_membership.dropdown-menu > li.newItem": "showModal"
+    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler"
```

Comment:
Probably. I didn't check. But I meant it in a general way.
Sorry, I remove my previous comment, I was that I was commenting the close PR. Maybe let's continue this disussion in the other one?